server/backends/blobstore/blobstore.go

```diff
@@ -224,7 +226,7 @@ func (d *DiskBlobStore) blobPath(blobName string) (string, error) {
 	if strings.Contains(blobName, "..") {
 		return "", fmt.Errorf("blobName (%s) must not contain ../", blobName)
 	}
-	return filepath.Join(d.rootDir, blobName), nil
+	return filepath.Join(*pathPrefix, d.rootDir, blobName), nil
```

Comment:
The `pathPrefix` should go after `rootDir` here; probably would be clearer to use `blobPath()` to be consistent with how we construct the paths for the other blobstores.

```suggestion
	return filepath.Join(d.rootDir, blobPath(blobName)), nil
```

Reply:
nice catch